pkg/cmd/taskrun/describe.go

```diff
@@ -32,7 +32,7 @@ import (
 
 const templ = `Name:	{{ .TaskRun.Name }}
 Namespace:	{{ .TaskRun.Namespace }}
-Task Ref:	{{ .TaskRun.Spec.TaskRef.Name }}
+Task Ref:    {{ taskRefExists .TaskRun.Spec }}
```

Comment:
would it be better to just skip showing `Task Ref:` label if the task ref is not present  (instead of showing that it isnt
) ?

Reply:
Yeah, agreed. We can just test if the result of `taskRefExists` is an empty string and show nothing if it's blank.